src/config.ts

```diff
@@ -123,6 +123,31 @@ const NodeList = async (): Promise<NodeInfo[]> => {
   return nodeList;
 };
 
+const NonStaleNodeList = async (
+  nodeList: NodeInfo[],
+  staleThreshold: number
+): Promise<NodeInfo[]> => {
+  return nodeList
+    .sort((a, b) => b.tip - a.tip)
+    .filter((node) => node.tip <= nodeList[0].tip - staleThreshold);
+};
+
+const clientWeightedSelector = async (
```

Comment:
```suggestion
const clientWeightedSelector = (
```

Doesn't need to be async.